src/core/storage.h

```diff
@@ -527,6 +527,22 @@ class Storage : public QObject
                                                      Message::Types type = Message::Types{-1},
                                                      Message::Flags flags = Message::Flags{-1}) = 0;
 
+    //! Request a certain number messages stored in a given buffer, matching certain filters, ascending
+    /** \param buffer   The buffer we request messages from
+     *  \param first    if != -1 return only messages with a MsgId >= first
+     *  \param last     if != -1 return only messages with a MsgId < last
+     *  \param limit    if != -1 limit the returned list to a max of \limit entries
+     *  \param type     The Message::Types that should be returned
```

Comment:
Minor overlooked matter - the `flags` parameter.

```suggestion
     *  \param type     The Message::Types that should be returned
     *  \param flags     The Message::Flags that should be returned
```